Source/i_main.c

```diff
@@ -77,6 +104,14 @@ static void I_Signal(void)
     }
 }
 
+//
+// D_DoomMain()
+// Not a globally visible function, just included for source reference,
+// calls all startup code, parses command line options.
+//
+
+void D_DoomMain(void);
```

Comment:
Why is this here?

Reply:
I removed the `d_main.h` header. The idea is to use `i_main.c` in the setup and in the main game like in Choco.

Reply:
Ack. 👍